nextflow.config

```diff
@@ -2,12 +2,19 @@ process {
   executor = 'slurm'
   time = '10m'
   clusterOptions = {
-    '-A b2015110 --qos=short -p node'
+    '-A b2015110 --qos=short -p core'
   }
 }
 
 params {
     modules {
-        samtools = 'samtools/1.3'
+        samtools = 'samtools/0.1.19'
+        manta = 'manta/0.27.1'
     }
+    programs {
+        svvcf2bed = "/home/rajohvik/Work/wgs-structvar/SVvcf2bed.pl"
+    }
+    threads = 16
+    ref_fasta = "/sw/data/uppnex/ToolBox/ReferenceAssemblies/hg38make/bundle/2.8/b37/human_g1k_v37.fasta"
+    //ref_fasta = "/sw/data/uppnex/reference/Homo_sapiens/g1k_v37/program_files/bwa/concat.fa"
```

Comment:
Maybe good to comment what this is, why it is commented out and/or when it is supposed to be used?